app/assets/stylesheets/common/components/welcome-header.scss

```diff
@@ -1,24 +1,35 @@
 .login-welcome-header {
+  box-sizing: border-box;
   display: grid;
   grid-template-columns: auto 1fr;
   grid-template-rows: auto 1fr;
   overflow-wrap: anywhere;
 
   .login-title {
-    font-size: var(--font-up-6);
     margin: 0;
     line-height: var(--line-height-medium);
+    font-size: var(--font-up-6);
+    @include breakpoint(mobile-extra-large) {
```

Comment:
btw, I see we use both `breakpoint("mobile-extra-large")` and `breakpoint(mobile-extra-large)` - we should probably decide which syntax is recommended.

(I'd love to also get a scss linter running at some point in a distant future 😄)